opentracing-testbed/src/test/java/io/opentracing/testbed/activate_deactivate/README.md

```diff
@@ -8,7 +8,8 @@ This example shows the usage of the `Continuation` concept (now part of the `Aut
 return new Thread(new Runnable() {
     @Override
     public void run() {
-	try (Scope scope = tracer.buildSpan("parent").startActive(false)) {
+	Span span = tracer.buildSpan("parent").start();
+	try (Scope scope = tracer.activateSpan(span)) {
```

Comment:
I thought the span must be manually finished now.  (There are other cases of this same problem elsewhere in this PR too.)

Reply:
Hey Tyler - the *activate deactivate* and *multiple callbacks* examples use the more exoteric `AutoFinishScopeManager`, which was actually ignoring the `finishSpanOnClose` parameter anyway, doing the `Span.finish()` call by itself eventually.

I think we could keep this examples along with this specific `ScopeManager` implementation for now, as it shows a very, very specific case (having to finish a `Span` being active in more than one thread/callback). Observe this implementation has its specific and well-known limitations and, IIRC, was kept for the `opentracing-java-v030` shim layer and also for potential usage along with `Scala` frameworks, given their callback nature.

Another options is to refactor the `AutoFinishScopeManager` code to have a similarity with the new API in all ways (for example, by adding a new method that does the conditional finish in the `Span`, maybe `Scope.Continuation.finishSpan()` or similar, deprecating the current API there). *If* we do this part, I'd suggest leaving it for another PR, as I think it's time to merge the current one (I can propose myself to do that change).

Let me know ;)

Reply:
Ah, I didn't notice the distinction in context here.  Carry on.